source/faq.rst

```diff
@@ -216,5 +216,63 @@ You can change the minimum resolution for metrics by the following way:
    when |pmm-server| and |pmm-client| are on different networks,
    or when :ref:`pmm.amazon-rds`.
 
+.. _alertmanager:
+
+How to integrate Alertmanager with PMM?
```

Comment:
if this is an FAQ the question should be the most relevant to a wide audience...so this question feels strange...how about "How do I integrate more robust alerting than Grafana native alerts?" or "Is PMM compatible with Prometheus Alertmanager?" or even "Does PMM support Alertmanager?"

This entire blurb is informational but doesn't really answer a question...just describes it...feels more like normal documentation than a frequently asked question.  I'll look at that PR next.